stock_manual_shipping_weight/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Roger Sans <roger.sans@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Stock Manual Shipping Weight',
+    'version': '14.0.1.0.0',
+    'author': 'Sygel',
+    'category': 'Stock',
+    'summary': 'The stock picking model has been modified to enable manual editing of the shipping weight field.',
```

Comment:
I think the summary can be simplified, something like "Manual edition of the shipping weight field in pickings".

Reply:
Done